Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM ubuntu:20.04
```

Comment:
How about using namely/docker-protoc to save a lot of build time?

Reply:
And setting environment GRPCIO_SYS_USE_PKG_CONFIG to 1 will force rust to link to share library grpc too, which can also save build time further.

Reply:
I tried namely/docker-protoc previously and ran into complications. This PR matches our github workflows which made it relatively easy to get working.

Reply:
I tried adding the env variable you recommend and then the build errors with ` thread 'main' panicked at 'can't find library grpc via pkg-config`. I am quite happy with this PR as is but would welcome future PRs that improve things further.

Reply:
A way to save docker image build time the best way would be to publish this image on dockerhub. I don't know who to contact about that.